src/components/tagline.js

```diff
@@ -1,11 +1,14 @@
 import React from "react";
 import { GridContainer, Grid } from "@trussworks/react-uswds";
 
-const Tagline = () => {
+const Tagline = ({taglineHeading, taglineText}) => {
```

Comment:
I'm assuming more props will be added that make it different from the `Section` component, but right now, they seem like the same component.

Reply:
Section and tagline are different if you look at the uswds landing page. They'll have slightly different layout